includes/admin/class-ur-admin-user-list-manager.php

```diff
@@ -56,7 +56,7 @@ public function __construct() {
 	 *
 	 * @return array
 	 */
-	public function ceate_quick_links( $actions, $user ) {
```

Comment:
@iamprazol if you are renaming function then you have to deprecate the previous function.